proposals/distribution.md

```diff
@@ -0,0 +1,66 @@
+# Abstract
+
+The Docker registry protocol has become the defacto standard across the container registry world ([https://github.com/docker/distribution/blob/master/docs/spec/api.md](https://github.com/docker/distribution/blob/master/docs/spec/api.md)).
+
+In the OCI, having a solid, common distribution specification with conformance testing will ensure long lasting security and interoperability throughout the container ecosystem.
+
+## Proposal
+
+TL;DR; Move [https://github.com/docker/distribution/tree/master/docs/spec](https://github.com/docker/distribution/tree/master/docs/spec) to [https://github.com/opencontainers/distribution-spec](https://github.com/opencontainers/distribution-spec)
+
+This proposal covers the distribution API spec, and while it does not cover the code for the docker-registry, that implementation is considered the reference implementation. There are other implementations of this protocol, not all are open-source though (Google gcr.io, Amazon ECR, CoreOS Quay, Gitlab registry, JFrog Artifactory registry, Huawei Dockyard, etc).
+
+In the past when the topic of having an OCI specification around the distribution of container images was discussed, it was deferred as "let’s get the image format defined, meanwhile the industry will settle on a distribution standard". Fast forward, OCI image format is out and adopted, and the Registry v2 is the defacto standard. There is and will be use-cases for alternate methods and the future will likely hold creative ways to push, fetch and share container images, but right now this promotion serves to acknowledge by the OCI the current industry standard of distributing container images.
+
+There is polish that is needed e.g. broken links to storage-driver docs, as well as making sections more generic regarding the OCI descriptors and media-types, but on the whole this is a lateral move.
+
+## Initial Maintainers
+
+* Stephen Day <stephen.day@docker.com> (@stevvooe)
+* Vincent Batts <vbatts@redhat.com> (@vbatts)
+* Derek McGowan <derek.mcgowan@docker.com> (@dmcgowan)
```

Comment:
With this proposal referencing project-template for the project's initial rules, we'll need a way to deal with project-template's [Chief Maintainer][1] role.  Possibilities include:

* Having the TOB select a Chief Maintainer here before putting this proposal up for a vote.
* Landing opencontainers/project-template#20 or some other project-template work that removes the Chief Maintainer role from the project docs before putting this proposal up for a vote.

My recommendation would be to remove the role (slightly more discussion [here][2] and [here][3]), but with that PR has been dangling for 1.4 years, so I don't know how likely it is to land in the next month.  Perhaps parallel work can land the Chief Maintainer removal, or opencontainers/project-template#41 may end up with the TOB in charge of project-template.  With project-template (currently) outside of direct TOB control, the simplest approach would be to appoint a Chief Maintainer here, as that entirely within the TOB's control.

[1]: https://github.com/opencontainers/project-template/blame/651c0ada9d70fed6036cf3e2403bf2ef0fb4df5d/MAINTAINERS_GUIDE.md#L78
[2]: https://github.com/opencontainers/project-template/pull/20#discussion_r95453349
[3]: https://github.com/opencontainers/runtime-spec/pull/420#discussion_r61972504

Reply:
> * Landing opencontainers/project-template#20 or some other project-template work that removes the Chief Maintainer role from the project docs before putting this proposal up for a vote.

This happened, so nothing left to do for this specific point.